### PR TITLE
Update TTN-Mapper to TTNv3

### PR DESCRIPTION
**Pull request by PaulReichmuth**
Maybe resize images, should be good to go otherwise

### CODE REVIEW

**Comment by mariopesch** [comment]
fixed the images @PaulReichmuth 
Could you update the programming section and use the following: https://blockly-react.netlify.app/gallery/5fd0a15c64d32d0011cab23a

This is a improved version of the ttn mapper as the cayenne payload is not really the best way for the ttn mapper

**Comment by mariopesch** [comment]
> Sure thing, can I use OTAA with that or does it have do be ABP?

@felixerdy both should work?

**Comment by felixerdy** [comment]
> > Sure thing, can I use OTAA with that or does it have do be ABP?
> 
> @felixerdy both should work?

Both should work but ABP is preferred in order to have synced GPS positions and LoRa transmission

**Comment by PaulReichmuth** [comment]
I might leave it at OTAA, with the extended info that ABP works better, so it's more newcomer-friendly. Switching activation methods in TTNv3 can get complicated pretty quick, so I'll stick with OTAA for now if that's alright

**Comment by PaulReichmuth** [comment]
@mariopesch updated code section

**Comment by mpfeil** [comment]
@mariopesch could you review this again and merge it asap.

File: _projects/de/2020-03-06-TTN-Mapper.md [hunk 1]
```diff
@@ -27,6 +17,19 @@ addons: ["LoRa-Bee"]
 lang: de
 tags: ["Geographie", "Informatik", "LoRa", "TTN", "Blockly"]
```
Comment (mpfeil) [inline _projects/de/2020-03-06-TTN-Mapper.md:18]: @PaulReichmuth could you also add the tag `LoRaWAN`

**Review by mpfeil — CHANGES_REQUESTED** [review]

File: _projects/de/2020-03-06-TTN-Mapper.md (hunk 1 above)
Comment (PaulReichmuth) [reply _projects/de/2020-03-06-TTN-Mapper.md:18]: One sec, will do

Comment (PaulReichmuth) [reply _projects/de/2020-03-06-TTN-Mapper.md]: @mpfeil done, should be in now

**Review by mariopesch — APPROVED** [review]
Looks good now, but I didn't try if it really works

**Pull request merged**